### ISSUE TITLE
problem after update to weewx 4.6.2-1

### ISSUE THREAD

**Issue by peters77** [chaunceygardiner/weewx-loopdata#6]
Hi John,

I updated form 4.5.2 to weewx 4.6.2-1 and now my weatherboard doesn't work any more.
Here is what I get at startup:

```
Feb 13 11:23:52 weewx weewx[1665230] INFO user.loopdata: Service version is 2.0.b6.
Feb 13 11:23:52 weewx weewx[1665230] ERROR user.loopdata: Could not find target_report: WeatherBoardReport.  LoopData is exiting. Exception: missing option "minute_label" in interpolation. 
```
The weatherboard wasn'tudated any more...my loop-data.txt file (place on /devs/shm) wasn't updated any more. 
Ther ssems to be a file generaterd but not copied by the report engine?

`Feb 13 12:00:45 weewx weewx[1667605] DEBUG weewx.reportengine: Running report 'WeatherBoardReport'
Feb 13 12:00:45 weewx weewx[1667605] DEBUG weewx.reportengine: Found configuration file /etc/weewx/skins/WeatherBoard/skin.conf for report 'WeatherBoardReport'
Feb 13 12:00:45 weewx weewx[1667605] DEBUG weewx.cheetahgenerator: Using search list ['weewx.cheetahgenerator.Almanac', 'weewx.cheetahgenerator.Current', 'weewx.cheetahgenerator.DisplayOptions', 'weewx.cheetahgenerator.Extras', 'weewx.cheetahgenerator.Gettext', 'weewx.cheetahgenerator.JSONHelpers', 'weewx.cheetahgenerator.PlotInfo', 'weewx.cheetahgenerator.SkinInfo', 'weewx.cheetahgenerator.Station', 'weewx.cheetahgenerator.Stats', 'weewx.cheetahgenerator.UnitInfo', 'user.dayboundarystats.DayboundaryStats']
Feb 13 12:00:45 weewx weewx[1667605] DEBUG weewx.manager: Daily summary version is 4.0
Feb 13 12:00:45 weewx weewx[1667605] INFO weewx.cheetahgenerator: Generated 1 files for report WeatherBoardReport in 0.02 seconds
Feb 13 12:00:45 weewx weewx[1667605] INFO weewx.reportengine: Copied 0 files to /var/www/html/weewx/weatherboard`

Any idea howto fix this!?

Regards,

Christian

**Comment by chaunceygardiner** [comment]
I have not yet tried 4.6.x.  It is early days and problems are still being reported.

Please try changing target_report to SeasonsReport in the [LoopData] section of weewx.conf.  Does LoopData successfully start with that change?

Then try changing the target_report to LoopDataReport.  Does LoopData successfully start with that change?

Please let me know what you find.  I’ll try to find time to try 4.6 soon.  I’ve been avoiding it because so much has changed that I’d rather look at it after most of the problems are fixed.

**Comment by peters77** [comment]
Yes, you are right...no good idea to try within this erarly stage of 4.6.x. 
I'd had some time and wanted to give it another try. As it works 99% (some problems arise, but the website builds up) I stay now for some hours on the new version, let's see.

 setting target_report to SeasonsReport reports:

`Feb 13 17:13:38 weewx weewx[1692711] INFO user.loopdata: Service version is 2.0.b6.
Feb 13 17:13:38 weewx weewx[1692711] ERROR user.loopdata: Could not find target_report: SeasonsReport.  LoopData is exiting. Exception: missing option "minute_label" in interpolation.
`

setting target_report to LoopDataReport

`Feb 13 17:14:52 weewx weewx[1692762] INFO user.loopdata: Service version is 2.0.b6.
Feb 13 17:14:52 weewx weewx[1692762] ERROR user.loopdata: Could not find target_report: LoopDataReport.  LoopData is exiting. Exception: 'LoopDataReport'`

Should I maybe first install your most recent version as I am on 2.0.b6?

**Comment by chaunceygardiner** [comment]
Thanks for getting back to me.  No, don’t install 2.0.b6.  I’ll load up 4.6 today and give this a try.  Loading reports has clearly changed (I’m sure because of all of the localization work).  I’ll likely have a fix for you to try sometime today.  Thanks for reporting this.

**Comment by peters77** [comment]
Thank you very much, don't feel stressed. I'll wait till you get time to have a look.

**Comment by chaunceygardiner** [comment]
Hi Christian,

I just published LoopData v2.9 which fixes the issue for WeeWX 4.6.x and is also backwards compatible with earlier versions of WeeWX 4.  Please give it a try.

Cheers,
John

**Comment by peters77** [comment]
John,

fixed! My WeatherBoard is online again.
Thanks for this ultra quick fix!!!

Regards,

Christia
